rajawali/src/main/java/org/rajawali3d/materials/Material.java

```diff
@@ -1073,7 +1073,12 @@ public void setModelMatrix(Matrix4 modelMatrix) {
         mModelMatrix = modelMatrix;//.getFloatValues();
         mVertexShader.setModelMatrix(mModelMatrix);
 
-        mNormalMatrix.setAll(modelMatrix).setToNormalMatrix();
+        mNormalMatrix.setAll(modelMatrix);
+        try {
+            mNormalMatrix.setToNormalMatrix();
+        } catch (IllegalStateException exception) {
+            // modelMatrix is degenerate (zero scale)
```

Comment:
Lets print a message here. Given the use case, I think a `RajLog.d()` is fitting so it only surfaces if debug output is enabled.